justfile

```diff
@@ -130,9 +144,12 @@ dart-test: rust-build dart-build
     dart test
 
 # Tests rust
-rust-test:
-    cd "$RUST_WORKSPACE"; \
-    cargo test --locked
+rust-test: rust-build
```

Comment:
is there something that will run with `cargo build` that will not with `cargo clippy`? My understanding was that we are covered when not in `cfg(test)` because is checked from clippy

Reply:
yes

Reply:
So we don't need `cargo build` even if it wans't run in `dart test` because of `cargo clippy`